index.bs

```diff
@@ -7155,6 +7155,45 @@ partial dictionary MLOpSupportLimits {
     ::
         The interpolation algorithm used to fill the output tensor values.
 
+        Both algorithms start with these inputs, computed for each spatial axis:
+        ```
+        scale = outputSize / inputSize
+        unclampedCoordinate = (outputCoordinate + 0.5) / scale - 0.5
+        inputCoordinate = clamp(unclampedCoordinate, 0, inputSize - 1)
+        ```
+        For a given `outputCoordinate.x` and `outputCoordinate.y` location in the output tensor, the above equations give a rational `inputCoordinate.x` and `inputCoordinate.y`.
+
+        <dl dfn-type=enum-value dfn-for=MLInterpolationMode>
+        : <dfn>nearest-neighbor</dfn>
+        ::
+            The `inputCoordinate.x` and `inputCoordinate.y` computed above are used as inputs to a nearest-neighbor sampling algorithm to compute the output tensor value as follows:
+            ```
+            x = ceil(inputCoordinate.x - 0.5)
+            y = ceil(inputCoordinate.y - 0.5)
+            output tensor value = input tensor value at (x, y)
+            ```
+
+        : <dfn>linear</dfn>
+        ::
+            The `inputCoordinate.x` and `inputCoordinate.y` computed above are used as inputs to a bilinear sampling algorithm to compute the output tensor value as follows:
+            ```
+            x0 = floor(inputCoordinate.x)
+            x1 = ceil(inputCoordinate.x)
+            y0 = floor(inputCoordinate.y)
+            y1 = ceil(inputCoordinate.y)
+            vx0y0 = input tensor value at (x0, y0)
+            v10 = input tensor value at (x1, y0)
+            v01 = input tensor value at (x0, y1)
```

Comment:
```suggestion
            vx0y1 = input tensor value at (x0, y1)
```

Reply:
Fixed in 8a34038 but please double check.